packages/frontend/app/lib/api/tenant.server.ts

```diff
@@ -0,0 +1,84 @@
+import { gql } from '@apollo/client'
```

Comment:
If you merge update `2893/multi-tenancy-v1` locally, you should see changes for this file from #3256